backend/app/api/routes/fine_tuning.py

```
@@ -0,0 +1,309 @@
+@router.post(
+    "/fine_tune",
+    description=load_description("fine_tuning/create.md"),
+    response_model=APIResponse,
+)
+def fine_tune_from_CSV(
+    session: SessionDep,
+    current_user: CurrentUserOrgProject,
+    request: FineTuningJobCreate,
+    background_tasks: BackgroundTasks,
+):
+    client = get_openai_client(  # Used here only to validate the user's OpenAI key;
```

Ruff (0.12.2)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)